extensions/terminal-suggest/src/completions/index.d.ts

```diff
@@ -1,1300 +1,1300 @@
+		 * - folders: show folders only. Allow autoexecute on folders
+		 * - history: show suggestions for all items in history matching this pattern
+		 * - help: show subcommands. Only includes the 'siblings' of the nearest 'parent' subcommand
+		 *
+		 * @example
+		 * `cd` uses the "folders" template
+		 * @example
+		 * `ls` uses  ["filepaths", "folders"]. Why both? Because if I `ls` a directory, we want to enable a user to autoexecute on this directory. If we just did "filepaths" they couldn't autoexecute.
+		 *
+		 */
+		template?: Template;
+		/**
+		 *
+		 * A function to filter and modify suggestions returned by a template
+		 *
+		 * @param templateSuggestions - the array of suggestion objects returned by the given template.
+		 * @returns An array of `Suggestion` objects.
+		 *
+		 * @example
+		 * The python spec has an arg object which has a template for "filepaths". However, we don't want to suggest non `.py` files. Therefore, we take the output of the template, filter out all files that don't end in `.py`, keep all folders that end with `/` and return the list of suggestions.
+		 */
+		filterTemplateSuggestions?: Function<TemplateSuggestion[], Suggestion[]>;
+		/**
+		 *
+		 * The command you wish to run on the user's device at their shell session's current working directory.
+		 *
+		 * @remarks
+		 * You can either specify
+		 * 1. a command and args to be executed (like `["ls"]` or `["git", "branch"]`)
+		 * 2. a function to generate the command and args to be executed. The function takes in an array of tokens of the user input and should output a array of string (command and args). You use a function when the script you run is dependent upon one of the tokens the user has already input (for instance an app name, a Kubernetes token etc.)
+		 * After executing the script, the stdout output will be passed to one of `splitOn` or `postProcess` for further processing to produce suggestion objects.
+		 *
+		 * @example
+		 * `git checkout <branch>` takes one argument which is a git branch. Its arg object has a generator with a `script: ["git", "branch"]"`. The stdout output of this shell command is then passed into the postProcess function to generate the final suggestions.
+		 */
+		script?:
+		| string[]
+		| Function<string[], string[]>
+		| ExecuteCommandInput
+		| Function<string[], ExecuteCommandInput>;
+		/**
+		 * Set the execution timeout of the command specified in the `script` prop.
+		 * @defaultValue 5000
+		 */
+		scriptTimeout?: number;
+		/**
+		 *
+		 * Process the string output from the `script` prop and return a list of suggestions
+		 *
+		 * @param out - The output of the script that was executed on the user's device
+		 * @param tokens - a tokenized array of what the user has typed
+		 * @returns An array of `Suggestion` objects.
+		 *
+		 */
+		postProcess?: (out: string, tokens: string[]) => (Suggestion | null)[]; // <-- VS Code edit to make results correct
```

Comment:
@meganrogge I added these comments to the 2 lines where we diverge to we can more easily track it